src/openzaak/client.py

```diff
@@ -3,10 +3,8 @@
 """
 Provide utilities to interact with other APIs as a client.
 """
-from typing import IO, Dict, Optional
-
-from zds_client.registry import registry
-from zgw_consumers.client import UnknownService, ZGWClient
+# TODO: remove legacy import of ZGWClient
```

Comment:
when will we replace `ZGWClient` with `ape_pie.APIClient`, in another PR?

Reply:
Yes I think that that should be a focus of another pr. Because these changes are explicitly to resolve some security issues. This won't change how anything operates so I think that doing this for this pr is fine.